blockchain/utxocache.go

```diff
@@ -71,6 +71,9 @@ type UtxoCacher interface {
 	// cache are removed from the provided view.
 	Commit(view *UtxoViewpoint) error
 
+	// FetchBackendState returns the current state of the UTXO set in the backend.
+	FetchBackendState() (*utxoSetState, error)
```

Comment:
Same deal here.  This is part of an interface, so the returned type should be exported as it would otherwise be unusable outside of the package.

Reply:
Updated.